src/Sparrow/LowMemory/CheckPageFileOnHdd.cs

```diff
@@ -397,9 +397,7 @@ public static string PosixIsSwappingOnHddInsteadOfSsd()
                         continue; // we do not check swap file, only partitions
 
                     // remove numbers at end of string (i.e.: /dev/sda5 ==> sda)
-                    var disk = swaps[i].DeviceName != null
-                        ? _regExRemoveNumbers.Replace(swaps[i].DeviceName, "").Replace("/dev/", "")
-                        : _regExRemoveNumbers.Replace("/dev/", "");
+                    var disk = _regExRemoveNumbers.Replace(swaps[i].DeviceName, "").Replace("/dev/", "");
```

Comment:
Do we have here swaps[i].DeviceName null error handling?

Reply:
We can't get null at `DeviceName` as it implemented now and we shouldn't. 
I added a test to check it.
I think if we still want to check this it should be by `Debug.Assert`

Reply:
PR Approved